### PR TITLE
slider

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please review it**
slider

### CODE REVIEW

File: Components/Slider/slider.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ApnaUI|Slider</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <link rel="stylesheet" href="slider.css">
+</head>
+<body>
+
+    <div class="slidercontainer">
```
Comment (NeerajVodala) [inline Components/Slider/slider.html:14]: recommended naming convention would be slider-container

File: Components/Slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+@import url("/layout.css");
+
+.slidercontainer {
+    width: 20rem;
+    box-shadow: 0px 0px 3px 1px  rgba(0,0,0,0.1);
+    height: 10rem;
+padding: 0.5rem 0 1rem 3rem;
```
Comment (NeerajVodala) [inline Components/Slider/slider.css:7]: consistent spacing would be recommended

**Review by NeerajVodala — COMMENTED** [review]
recommended changes (not mandatory): added two comments, please check. All the best

File: Components/Slider/slider.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  }
+  
+  .slider {
+    -webkit-appearance: none;
+    width: 10rem;
+    height: 25px;
+    background: #d3d3d3;
```
Comment (ravikumar0403) [inline Components/Slider/slider.css:14]: use variables for colors

File: Components/Slider/slider.html (hunk 1 above)
Comment (anshtripathi01) [reply Components/Slider/slider.html]: thanks fixed it

**Pull request merged**